README.md

```diff
@@ -12,36 +13,57 @@
 - Inference results from model endpoint are directly handled in `model_inference/inference.py`
 
 ****
- 
+
 ### RUNNING NACHET-BACKEND FROM DEVCONTAINER
-When you are developping, you can run the program while in the devcontainer by using this command:
-```
+
+When you are developping, you can run the program while in the devcontainer by
+using this command:
+
+```bash
 hypercorn -b :8080 app:app
 ```
 
 ### RUNNING NACHET-BACKEND AS A DOCKER CONTAINER
-If you want to run the program as a Docker container (e.g., for production), use: 
-```
+
+If you want to run the program as a Docker container (e.g., for production), use:
+
+```bash
 docker build -t nachet-backend .
 docker run -p 8080:8080 -v $(pwd):/app nachet-backend
 ```
 
 ### TESTING NACHET-BACKEND
-To test the program, use this command: 
-```
+
+To test the program, use this command:
+
+```bash
 python -m unittest discover -s tests
 ```
 
 ****
+
 ### ENVIRONMENT VARIABLES
-Start by making a copy of `.env.template` and renaming it `.env`. For the backend to function, you will need to add the missing values:
 
-* **NACHET_AZURE_STORAGE_CONNECTION_STRING**: Connection string to access external storage (Azure Blob Storage).
-* **NACHET_MODEL_ENDPOINT_REST_URL**: Endpoint to communicate with deployed model for inferencing. 
-* **NACHET_MODEL_ENDPOINT_ACCESS_KEY**: Key used when consuming online endpoint.
-* **NACHET_DATA**: Url to access nachet-data repository
-* **NACHET_HEALTH_MESSAGE**: Health check message for the server.
+Start by making a copy of `.env.template` and renaming it `.env`. For the
+backend to function, you will need to add the missing values:
+
+- **NACHET_AZURE_STORAGE_CONNECTION_STRING**: Connection string to access
+  external storage (Azure Blob Storage).
+- **NACHET_MODEL_ENDPOINT_REST_URL**: Endpoint to communicate with deployed
+  model for inferencing.
+- **NACHET_MODEL_ENDPOINT_ACCESS_KEY**: Key used when consuming online endpoint.
+- **NACHET_DATA**: Url to access nachet-data repository
+- **NACHET_SUBSCRIPTION_ID**
+- **NACHET_RESOURCE_GROUP**
+- **NACHET_WORKSPACE**
+- **NACHET_MODEL**
+- **NACHET_BLOB_PIPELINE_NAME**
+- **NACHET_BLOB_PIPELINE_VERSION**
+- **NACHET_BLOB_PIPELINE_DECRYPTION_KEY**
```

Comment:
For consistency, add more descriptive details.

Reply:
Done